index.html

```diff
@@ -1328,6 +1325,114 @@
         </section>
         <section>
           <h4>
+            Event Handlers
+          </h4>
+          <p>
+            The following are the event handlers (and their corresponding event
+            handler event types) that must be supported, as event handler IDL
+            attributes, by objects implementing the <a>PresentationRequest</a>
+            interface:
+          </p>
+          <table dfn-for="PresentationRequest">
+            <thead>
+              <tr>
+                <th>
+                  Event handler
+                </th>
+                <th>
+                  Event handler event type
+                </th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr>
+                <td>
+                  <dfn><code>onconnect</code></dfn>
+                </td>
+                <td>
+                  <code>connect</code>
+                </td>
+              </tr>
+            </tbody>
+          </table>
+        </section>
+        <section>
+          <h3>
+            Interface <code>PresentationSessionConnectEvent</code>
+          </h3>
+          <pre class="idl">
+            [Constructor(DOMString type, optional PresentationSessionConnectEventInit eventInitDict)]
+            interface PresentationSessionConnectEvent : Event {
+              [SameObject] readonly attribute PresentationSession session;
+            };
+
+            dictionary PresentationSessionConnectEventInit : EventInit {
+              PresentationSession session;
+            };
+          
+</pre>
+          <p>
+            An event named <code>connect</code> is fired on a
+            <a>PresentationRequest</a> when a session associated with the
+            object is initiated. It is fired at the <a>PresentationRequest</a>
+            instance, using the <a>PresentationSessionConnectEvent</a>
+            interface, with the <a for=
+            "PresentationSessionConnectEvent">session</a> attribute set to the
+            <a><code>PresentationSession</code></a> object that was initiated.
+            The event is fired regardless of how the session was initiated. For
+            example, it can come from a call to <code>start()</code>,
```

Comment:
We should make it clear that the connect event is fired as soon as possible, i.e. as soon as the user selects a device, so that the page can update appropriately. For example, if the page is playing a video it would want to stop that immediately and indicate to the user that their request to fling the video to the device is in progress.